include/openthread/thread.h

```diff
@@ -396,14 +419,16 @@ const otNetworkKey *otThreadGetNetworkKey(otInstance *aInstance);
  * call to this function invalidates the Active and Pending Operational Datasets in
  * non-volatile memory.
  *
+ * The function expects the user to pass the thrNetworkKey as a hex literal.
+ *
```

Comment:
I believe in future we could extend this function with `otCryptoType` parameter to set key using reference or its value.